isis/CMakeLists.txt

```diff
@@ -283,9 +314,6 @@ endforeach()
 foreach (_variableName ${_variableNames})
    get_filename_component(LIBDIR "${${_variableName}}" DIRECTORY)
    if (_variableName MATCHES "^CMAKE+")
-   elseif (_variableName MATCHES ".+_LIB$")
-     list(APPEND ALLLIBDIRS "${LIBDIR}")
-     list(APPEND ALLLIBS "${${_variableName}}")
```

Comment:
This if-statement wasn't really being used before, after switching to stock FindBoost.cmake, it caused vars internal to that script to slip through, so I simply removed it.